agent/main/lib/Worker.ts

```diff
@@ -154,8 +160,8 @@ export class Worker extends TypedEventEmitter<IWorkerEvents> implements IWorker
 
   private resumeAfterEmulation(): Promise<any> {
     return Promise.all([
-      this.devtoolsSession.send('Runtime.runIfWaitingForDebugger'),
       this.devtoolsSession.send('Debugger.disable'),
+      this.devtoolsSession.send('Runtime.runIfWaitingForDebugger'),
```

Comment:
I think this might have actually needed to be before the disable. Did you find it needed to go after? I think I had timing issues occasionally